module/VuFindSearch/src/VuFindSearch/Backend/EDS/SearchRequestModel.php

```diff
@@ -66,6 +67,13 @@ class SearchRequestModel
      */
     protected $facetFilters = [];
```

Comment:
$facetFilters is still an array, but the structure has changed significantly in a backwards-incompatible way.  Only relevant if someone actually extends this.

Reply:
Yeah, I think it's okay to include this in a point release with a changelog note due to the likelihood that nobody has ever had any reason to customize it. :-)